src/modules/polls.ts

```diff
@@ -233,28 +229,30 @@ export async function handleCommand(
 			try {
 				const id = interaction.options.getString("id", true);
 				const title = interaction.options.getString("title", true);
-				const cron = interaction.options.getString("cron", true);
 				const channel = interaction.options.getChannel("channel", true);
+				const cron = interaction.options.getString(
+					"schedule",
+					false
+				) as string | null;
```

Comment:
`getString(..., false)` already returns a `string | null`; annotation is unnecessary
```suggestion
				);
```

Reply:
:ok_hand: